RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -372,13 +373,14 @@
      qualityCuts = [-0.6,-0.45,-0.3]
 ))
 
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackdnn.toReplaceWith(tobTecStep, TrackLwtnnClassifier.clone(
-     src         = 'tobTecStepTracks',
-     qualityCuts = [-0.4, -0.25, -0.1]
+from RecoTracker.FinalTrackSelectors.TrackTfClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
+trackdnn.toReplaceWith(tobTecStep, TrackTfClassifier.clone(
+     src = 'tobTecStepTracks',
+     qualityCuts = qualityCutDictionary["TobTecStep"]
 ))
-(trackdnn & fastSim).toModify(tobTecStep,vertices = 'firstStepPrimaryVerticesBeforeMixing')
```

Comment:
this change with the whitespace and quotes is not strictly needed, please revert